velox/functions/prestosql/tests/ArrayIntersectTest.cpp

```diff
@@ -735,3 +735,50 @@ TEST_F(ArrayIntersectTest, nullInDictionary) {
       makeNullableArrayVector<int64_t>({{{1}}, std::nullopt, {{7}}});
   testExpr(expected, "array_intersect(c0)", {arrays});
 }
+
+TEST_F(ArrayIntersectTest, unknownArrays) {
+  auto array1 = makeNullableArrayVector<UnknownValue>(
```

Comment:
This is a bit hard to read. Is there a way to extend makeArrayVectorFromJson to work with Unknown?